app/urls/api.py

```diff
@@ -1,10 +1,12 @@
+from django.contrib.auth import views as auth_views
 from django.urls import path
 
 from app.views import api
 
 urlpatterns = [
-    path("token/", api.AuthTokenView.as_view(), name="obtain-auth-token"),
-    path("refresh-token/", api.RefreshTokenView.as_view(), name="refresh-auth-token"),
+    path("is_authenticated/", api.CheckAuthenticatedView.as_view()),
```

Comment:
change it to "me" api instead, if you have already that api use that and remove this one.
we can fetch self user details if api is successful it means user is logged in